app/components/searchable_collection_component/searchable_collection_component.scss

```diff
@@ -23,7 +23,7 @@
     }
   }
 
-  .searchable-collection-component__search {
+  &__search {
```

Comment:
Is there a difference between `__` and `--` in these different class names?

Reply:
__ is a block, -- should signify a modifier (state). we dont use this to letter by any means but im trying to use them correctly particuarly in components

Reply:
it is an actual CSS convention, not something ive randomly invented, i should point out!